lte/gateway/c/core/oai/tasks/nas5g/src/AmfMessage.cpp

```diff
@@ -14,6 +14,7 @@
 #include "lte/gateway/c/core/oai/tasks/nas5g/include/AmfMessage.h"
 #include "lte/gateway/c/core/oai/tasks/nas5g/include/M5gNasMessage.h"
 #include "lte/gateway/c/core/oai/tasks/nas5g/include/M5GCommonDefs.h"
+#include "lte/gateway/c/core/oai/tasks/amf/amf_app_defs.h"
```

Comment:
Is this added to include `lte/gateway/c/core/oai/common/log.h`? In that case, include the log file directly, please.

Reply:
addressed